base_iso3166/models/res_country.py

```diff
@@ -29,13 +33,15 @@ class ResCountry(models.Model):
     def _compute_codes(self):
         for country in self:
```

Comment:
```suggestion
        for country in self.filtered("code"):
```

Reply:
A) in the case of code=False, it's not calculating  "code_alpha3" and "code_numeric" which will produce errors
-> I would need to handle this case outside of the loop which would make the code more complex.
or do I miss something?

Reply:
Do a previous `self.code_alpha3 = False`. And please squash everything in one commit.

Reply:
You are still not implementing this and changing the existing code, so CI has broken